### DISCUSSION TITLE
email links

### DISCUSSION THREAD

**Discussion by magicman32** [JulianPrieber/littlelink-custom#115] [General]
Sorry to be a pain,

when adding a link to email, whats the format in url?? for a gmail email for example

**Comment by JulianPrieber** [comment]
The format should just be: `someone@gmail.com`

**Reply by JulianPrieber** [reply]
Previously, you had to enter `mailto:someone@gmail.com` this has been changed now.
Although, old links added in this format should still work.

You should even be able to add a mail to a custom button for example, but there you still have to use the format: `mailto:someone@gmail.com`

**Reply by magicman32** [reply]
thank you

**Comment by magicman32** [comment]
on Unraid Docker only works with mailto:someone@gmail.com
